src/js/src/components/ToolboxComponents/DomainStats.vue

```diff
@@ -58,9 +58,13 @@
 
 import { requestService } from '../../services/RequestService'
 import domainScript from './ToolboxResources/domainStats'
+import StringManipulationUtils from './../../mixins/StringManipulationUtils'
 
 export default {
   name: 'DomainStats',
+
+  mixins: [StringManipulationUtils],
```

Comment:
_StringManipulationUtils_ not complying with our $_ naming convention for mixins.